chromoscores/scorefunctions.py

```diff
@@ -123,38 +188,52 @@ def peak_score(
 """Isolation score"""
 
 
-def isolation_score(contact_map, delta, diag_offset, max_dist, state, pseudo_count=1):
+def isolation_score(snippet, delta, diag_offset, max_dist, state, pseudo_count=1):
```

Comment:
suggestions:
- state --> change this to a string, isolation_area='square' or 'triangle

- set pseudo_count default to zero

its not clear what diag_offset vs. max_distance represent